apps/src/StudioApp.js

```diff
@@ -1039,12 +1047,8 @@ StudioApp.prototype.configureDom = function (config) {
 
   var runButton = container.querySelector('#runButton');
   var resetButton = container.querySelector('#resetButton');
-  var throttledRunClick = _.debounce(function () {
-    if (window.$) {
-      $(window).trigger('run_button_pressed');
```

Comment:
good catch! I can keep it as run_button_pressed. the reason was because the "view data" button would be triggering it too, and "save project" describes both usages where as "run button pressed" only describes one of two uses. I will change this back.